File: Programmazione e laboratorio/Tutor/Tutor9/Bibloteca/libro.c
#include "libro.h"
void inserimento_libro(Libro *b){
    printf("Inserisci il titolo del libro:\n");
    scanf("%s",b->titolo);
    printf("Inserisci l'autore del libro:\n");
    scanf("%s",b->autore);
    printf("Inserisci l'anno di pubblicazione del libro:\n");
    scanf("%d",&b->anno_pubblicazione);
    printf("\n");
}

File: Linguaggi_programmazione/esercizi/provaEsame03/Libro.java
public class Libro {
  private String titolo, autore;
  private int prezzo;

  public Libro(String titolo, String autore, int prezzo){
    this.titolo=titolo;
    this.autore=autore;
    this.prezzo=prezzo;
  }

  public String getTitolo(){
    return this.titolo;
  }

  public String getAutore(){
    return this.autore;
  }

  public int getPrezzo(){
    return this.prezzo;
  }
}
